src/librustc/diagnostics.rs

```diff
@@ -947,6 +945,8 @@ position that needs that trait. For example, when the following code is
 compiled:
 
 ```compile_fail
+#![feature(on_unimplemented)]
+
```

Comment:
Because to use `#[rustc_on_unimplemented]`, you need to enable the feature. However, the code throws `E0230`, so I can't add `E0272`. The same goes for `E0273` and `E0274`.